packages/expo-location/android/src/main/java/expo/modules/location/services/LocationTaskService.java

```diff
@@ -64,8 +64,13 @@ public void stop() {
 
   @Override
   public void onTaskRemoved(Intent rootIntent) {
-    super.onTaskRemoved(rootIntent);
-    stop();
+    Bundle extras = rootIntent.getExtras();
+    extras.getString("killService");
+    
+    if(killService == "true"){
+      super.onTaskRemoved(rootIntent);
```

Comment:
`super.onTaskRemoved(rootIntent);` should be called even if `killService` = `true` ?

Reply:
I don't think that's true. Normally on Android, when you have to call a super method, this method is annotated with `@CallSuper`.